app/src/main/java/org/jellyfin/androidtv/model/itemtypes/ImageCollection.kt

```diff
@@ -15,14 +16,28 @@ class ImageCollection(original: BaseItemDto) {
 	val backdrops = original.backdropImageTags.map { Image(original.id, ImageType.Backdrop, it) }.toList()
 
 	class Image(private val itemId: String, private val type: ImageType, private val tag: String) {
+		var index = 0
+
 		val url: String by lazy {
+			if (type == ImageType.Chapter)
+				TvApp.getApplication().apiClient.GetImageUrl(itemId, ImageOptions().also {
+					it.imageType = type
+					it.tag = tag
+					it.imageIndex = index
+				})
+			else
 			TvApp.getApplication().apiClient.GetImageUrl(itemId, ImageOptions().also {
 				it.imageType = type
 				it.tag = tag
```

Comment:
Just add the index here:
`if(imageIndex != null) it.index = imageIndex`

Might even be possible to do it without  the if statement (I think null is the default value)